src/base.config.ts

```diff
@@ -193,6 +194,24 @@ export default function webpackConfigFactory(args: any): WebpackConfiguration {
 		}
 	};
 
+	const postcssImportConfig = {
+		filter: (path: string) => {
+			return /.*variables(\.m)?\.css$/.test(path);
```

Comment:
This means the postcss-import with process imports that match either `variables.m.css` or `variables.css`